docs/configuration/translation-strings.md

```diff
@@ -0,0 +1,65 @@
+# Translation Strings
+
+> Translation Strings are multilingual key-value pairs that you can use throughout the app. They enable you to translate
+> things like dropdown options, placeholder text, field notes, and more.
+
+[[toc]]
+
+## How It Works
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220615A/translation-strings-20220615A.mp4" type="video/mp4" />
+</video>
+
+When creating a Field, you may want to add a note or other description along with it to clarify its context or purpose.
```

Comment:
> may want to add a note or other description along with it to clarify its context or purpose

This makes it seem as if this is the only purpose of Translation Strings. This is an open-ended feature, and these "multilingual variables" can be used in many places (and even more over time).

Reply:
I considered this. I was trying to focus on what it currently does, then just update the docs when we have a broader future use cases. I think making this more specific now adds some clarity to what purpose it serves as well as some contrast to the other translation features, but I will remove it.